.github/workflows/beta-release.yaml

```diff
@@ -10,33 +10,11 @@ permissions:
   id-token: write
   contents: read
 
-env:
-  IMAGE_HOST: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com"
-  IMAGE: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller"
-  IMAGE_TAG: "${{ github.event.inputs.tag }}"
-  IMAGE_TAG_AMD: "${{ github.event.inputs.tag }}-linux_amd64"
-  IMAGE_TAG_ARM: "${{ github.event.inputs.tag }}-linux_arm64"
-
 jobs:
-  integration-test:
-    name: Integration Test
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Checkout Code
-        uses: actions/checkout@v3
-        with:
-          ref: refs/tags/${{ github.event.inputs.tag }}
-      - name: Run integration test action
-        uses: ./.github/actions/integration-test
-        with:
-          aws_role: "${{ secrets.BETA_TEST_AWS_ROLE }}"
-          vpc_id: "${{ secrets.INTEG_TEST_VPC }}"
-          account_id: "${{ secrets.BETA_AWS_ACCOUNT }}"
 
   push-images:
     name: Build And Push Images
     runs-on: ubuntu-22.04
-    needs: [ integration-test ]
```

Comment:
how are we enforcing integration-tests before beta push and eventually prod push?

Reply:
Reverted. There are some process changes I want to make around testing for a release, but this isn't the right place to start them.